tests/api/cars.api.spec.js

```diff
@@ -0,0 +1,105 @@
+import { test, expect, request } from '@playwright/test';
+
+test.describe('API Tests for Cars', () => {
+  let apiContext;
+
+  test.beforeAll(async () => {
+    // Создаем новый контекст API с базовым URL
+    apiContext = await request.newContext();
+
+    // Авторизация пользователя
+    const loginResponse = await apiContext.post('api/auth/signin', {
+      data: {
+        email: 'oleksiihaidukqa@gmail.com',
+        password: 'Q123q123_',
+        remember: false,
+      },
+    });
+
+    if (!loginResponse.ok()) {
+      const responseText = await loginResponse.text();
+      console.error('Login failed:', responseText); // Выводим текст ответа, если ошибка
+      throw new Error(`Failed to login: ${responseText}`);
+    }
+
+    const loginResponseBody = await loginResponse.json().catch((err) => {
+      console.error('Error parsing JSON:', err);
+      throw err;
+    });
+
+    console.log('Login Response:', loginResponseBody);
+    expect(loginResponseBody.status).toBe('ok');
```

Comment:
you are not going to do it this way on real project

but let it be here for practice